packetnetworking/distros/debian/test_bonded.py

```diff
@@ -79,6 +80,24 @@ def test_public_bonded_task_etc_network_interfaces(
             post-down route del -net 10.0.0.0/8 gw {ipv4priv.gateway}
         """
     result += dedent(partial)
+
+    for bond, members in builder.network.bonds.items():
+        if bond == "bond0":
+            continue
+
+        partial = f"""
+            auto {bond}
+            iface {bond} inet static
+                bond-downdelay 200
+                bond-miimon 100
+                bond-mode {bonding_mode}
+                bond-updelay 200
+                bond-xmit_hash_policy layer3+4
+            """
+        result += dedent(partial)
+        if distro == "ubuntu":
```

Comment:
lacp-rate doesn't apply to debian also?

Reply:
it probably does but we want to handle debian/ubuntu disparity in a separate issue. Same for alpine.